core/shared/src/main/scala/com/monovore/decline/CommandApp.scala

```diff
@@ -22,7 +22,7 @@ class CommandApp(command: Command[Unit]) {
   }
 
   def main(args: Array[String]): Unit =
-    command.parse(args) match {
+    command.parse(PlatformApp.ambientArgs getOrElse args) match {
```

Comment:
I agree with @johnynek. During my experience I found that `CommandApp` is not really useful for me. Usually I end up with something like this:
```
val params: Params[XXX] = {
  ...
}
def run(xxx: XXX) = ???
new CommandApp(
      "xxx",
      "xxx",
      params.map(run)
    ).main(Array.empty)
```